lib/fluent/plugin/out_splunk_hec.rb

```diff
@@ -104,18 +107,27 @@ def format_event(time, record)
 
       if record[@host_key]
         msg['host'] = record[@host_key]
```

Comment:
`@remove_host_key ?  record.delete(@host_key) : record[@host_key]` is better

Reply:
Thanks, I'll update the PR

Reply:
updated